_slides/deepfried_dd.md

```diff
@@ -5,22 +5,21 @@ title: Deepfried_DD
 
 ## The Real dd
 
-* dd is a command-line utility used to convert and copy files (highly configurable)
-* You might recognize `dd if=/dev/zero of=/dev/null`
-* The real `dd` performs block level I/O, as opposed to filesystem-level I/O, for increased performance
-* As a result, if misused, it can corrupt your hard disk/SSD (be very careful running this as root)
-* You will be implementing this tool using file-level I/O (`fread`, `fwrite`, etc.)
+* A highly configurable command-line utility used to convert and copy files
+* Performs block level I/O, as opposed to filesystem-level I/O for increased performance
 
-## Applications of `dd`
+## What is a Block?
 
-* Data transfer, and in-place modification of files
-* Wiping disks for security (e.g. prior to recycling hardware)
-  - Contents of a file are not necessarily overwritten on regular deletion
-* Generating files with random data (will be useful for testing in upcoming Networking MP)
-* "Flashing" / installing custom ROMs (enhanced versions of vanilla OS) on Android devices
+* A unit measuring the number of bytes that are read or written at one time. 
```

Comment:
This definition of block is a bit imprecise. I would just call it “a sequence of bits with a predefined size ( the block size)”